.github/workflows/deploy-storybook.yml

```
@@ -0,0 +1,34 @@
+name: Deploy Storybook
+
+on:
+  push:
+    branches:
+      - "develop"
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.build-publish.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      # Manual Checkout
+      - uses: actions/checkout@v4
+
+      # Set up Node
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
+
+      # Add Storybook build and deploy to GitHub Pages
+      - id: build-publish
+        uses: bitovi/github-actions-storybook-to-github-pages@v1.0.3
+        with:
+          build_command: npm run storybook:build-wc
+          path: storybook-static/web-components
+          checkout: false
```

If we want to use built DDS library instead of source code, we can do as follows:
(requires Requires prior `npm ci`)

Given that there is no difference between using a built package and using source code if made correctly (since both DDS and Storybook are built with Vite), and given that `develop` commits are only created by PRs that have passed CI testing, this is perfectly *optional*.
Also, the execution time will be shorter if we *don't* build.

```suggestion
      - name: Build
        run: npm run build

      # Add Storybook build and deploy to GitHub Pages
      - id: build-publish
        uses: bitovi/github-actions-storybook-to-github-pages@v1.0.3
        with:
          build_command: npm run storybook:build-wc
          path: storybook-static/web-components
          checkout: false
```

---

@yodas7 Thank you for the suggestion - I will make this change